### ISSUE TITLE
Can anybody achieves the result of paper ?

### ISSUE THREAD

**Issue by phoebe0920** [laughtervv/DepthAwareCNN#30]
I use the  sunrgbd dataset  to train, and get the following results, but the  results are quite different from the experimental results of the paper.

> Accuracy: 52.18331861069419
 Accuracy Class: 21.65154302916059
 Mean IU: 13.361277218676214
FWAV Accuracy: 36.994343765004906

And I directly use the pretrain-Model(best_net_HHA.pth,best_net_rgb.pth) to test nyuv2, and also get the bad result as following:

>  Accuracy: 5.886450036442711
Accuracy Class: 2.6109730562398203
Mean IU: 0.697199549228473
FWAV Accuracy: 2.2405542009448167

Can anyone give me some advises,please? Thank you very much

**Comment by xieyuhaoli** [comment]
Excuse me, how did you get the result? Why did I encounter so many problems? May I ask you?
